corehq/apps/reports/static/reports/v2/js/columns.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * todo add docstring
+ */
+
+hqDefine('reports/v2/js/columns', [
+  'lodash',  // can we switch to lodash?
```

Comment:
I'm in favor, though I'm not in love with using both underscore and lodash simultaneously, it'd help if you could commit some amount of time to migrating (or organizing a team day to migrate - I don't think it's a difficult task, just a bit tedious). Curious what specifically you want lodash for?

Reply:
went back to using just `underscore` for now

Reply:
Thank you - this what I was most worried about this PR (potentially having both underscore and lodash). We'll get onto lodash at some point :)